micrometer-core/src/main/java/io/micrometer/core/instrument/docs/MeterDocumentation.java

```diff
@@ -105,7 +104,8 @@ default KeyName[] getAdditionalKeyNames() {
      * @return {@link MeterDocumentation} for which you don't want to create a default
      * metric documentation
      */
-    default Enum<?> overridesDefaultMetricFrom() {
+    // TODO should this be nullable?
+    default @Nullable Enum<?> overridesDefaultMetricFrom() {
         return null;
```

Comment:
I wasn't really sure how this is used and marking it `@Nullable` is probably fine. We can follow-up if we want to change it.